controller/error.go

```diff
@@ -6,6 +6,22 @@ import (
 	"github.com/juju/errgo"
 )
 
+// ValidationError capsules validation errors into one error struct.
+// It is returned when the validation fails. causingErrors contains all
+// errors that occurenced, while validating the request.
+type ValidationError struct {
+	causingErrors []error
+}
+
+func (e ValidationError) Error() string {
+	// TODO better error message
+	msg := "group or unit invalid:\n"
```

Comment:
How about we return the group as part of the validationError and print the errors by group?

```
Validation Errors found:
 * group 'mygroup': mixes scalable and unscalable units
 * group 'mygroup': wrong prefix
 * group 'global-simple-unit': wrong prefix
```